### PR TITLE
Added *lots* of german words

### PR DESCRIPTION
**Pull request by riker09**
I tried to watch out for profanity but I cannot guarantee that some fowl language has slipped through.

### CODE REVIEW

**Comment by killercup** [comment]
Thanks for the contribution! I don't really wanna replace the list of words I picked personally with a random list based on usage. If you think this is a category you'd use, can you add it as a new one? Maybe called "Common German words"?

**Comment by riker09** [comment]
Thanks for taking a look at my changes. I reverted your German Words category back and added my own. I definitly think that this is some I use since my team agreed on the naming schema "English Adjective" and "German Word" (preferrably a noun) together with "alliteration: on" for naming our releases. The motivation behind this was that with only a small list there is a lot of repitition.

File: categories.js [hunk 1]
```diff
@@ -478,7 +478,7 @@ var categories = {
       "hypochlorite",
       "bitartrate",
       "thiophosphate",
-      "crystal ",
+      "crystal ",
```
Comment (killercup) [inline categories.js]:
```suggestion
      "crystal",
```

Can you revert this space? :)

**Review by killercup — COMMENTED** [review]
Thanks for the quick update! One small thing and then it's good to go

File: categories.js (hunk 1 above)
Comment (riker09) [reply categories.js]: My editor must have done this by itself, I didn't touch that line and it must have slipped.

**Comment by killercup** [comment]
Thanks!

**Pull request merged**